test/support/page_objects/dashboard_page.js

```diff
@@ -14,13 +14,13 @@ export default class DashboardPage {
 
   clickNewDashboard() {
     return this.findTimeout
-    .findByCssSelector('button.ng-scope[aria-label="New Dashboard"]')
+    .findByCssSelector('[aria-label="New Dashboard"]')
```

Comment:
Making our test selectors too specific results in tests that are brittle and tightly-coupled to the DOM.